### ISSUE TITLE
AI Not Obediant

### ISSUE THREAD

**Issue by Joshuwat** [genesis92x/VcomAI-3.0#142]
**Describe the bug**
Started playing and messing around with the AI in a Zeus game. Set up an FOB in one of the missions I set up and placed a patrol that would be attacked. When attacked, the patrol called in every single friendly force for only around 7 OPFOR units. So, first problem is AI sends too many reinforcements for really small groups. Second, AI does not go back to where it originally was, just stays in one place. Seems like all this happens after the Contact DLC was released. AI also does not completely follow waypoints and they don't always do what they're told, such as land, fire mission, whatnot. AI does also go into each other sometimes, not much however.

**To Reproduce**
1. Just make a game with Zeus, AI, just a game
2. Get people to fight, with friendlies in the area, preferably a large FOB
3. Let the game play out
4. You'll understand what I'm saying

**Expected behavior**
AI is supposed to follow waypoints, go back to their original point or patrol on previous area, have adaptive reinforcements, not just a whole FOB with 4 helis and tens of units for a small squad. AI not into each other

**Screenshots**
Screenshots would not do it justice, needs to be seen by itself.

**VCOM Version:**
 - Branch: Master? Don't know where to find this
 - Version 3.3.3
 
**Additional mods used:**
No mods and with 40+ mods

**Additional context**
I may find other problems with the AI but I'm pretty sure it just needs a patch to work with Contact. Don't listen to me though, idk what I'm doing with coding and whatnot, that's your job.

**Comment by Joshuwat** [comment]
Forgot to mention, I get errors when I enable the advanced sniper AI whenever I have Marksmen, Snipers, or Sharpshooters in the map and they engage in combat. I will provide a picture to this one.  Happened for another error as well, however, I do not remember the reasoning for this.
https://imgur.com/4wh6juX

**Comment by genesis92x** [comment]
Looks like you are running CBA, the second posted error, as I found out on the bohemia forums, comes from CBA. Reload your CBA vcom settings and the issue will be resolved. For the primary issue, you need to change the settings to get expected behavior. That's your job as a mission maker.

**Comment by Joshuwat** [comment]
May I ask what you mean by reloading my CBA Vcom settings? Is that just turning it off and back on in the vcom options? I did that and see no difference

**Comment by Joshuwat** [comment]
Sorry to bother you with this; however, I have reloaded the CBA Vcom settings option and no changes have been made to the AI that I have seen. Massive reinforcements still come in and helicopters and cars still stay in one area after combat is over. The error message I get is now removed, but AI itself is still being buggy for me. I have also messed with the settings of the addon itself and I can't seem to get to lower the amount of reinforcements or make them adaptive to the Opposing Force, it just moves all units within the certain area. Is there a way to add that option or something like that to be able to change the reinforcements' size and not just their distance? And is there a way to fix the vehicles staying in one area after combat is over? Once again, I am extremely sorry to bother you with this, I would just like to get this fixed before my team starts our Mil Sims.
